src/main/java/cz/codecamp/logger/loggers/MultiLogger.java

```diff
@@ -0,0 +1,22 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import cz.codecamp.logger.LoggerInterface;
+import cz.codecamp.logger.PragmaticLoggerInterface;
+
+
+public class MultiLogger implements PragmaticLoggerInterface {
+
+    private LoggerInterface [] loggerInterface;
+
+    public MultiLogger(LoggerInterface... loggerInterface){
+        this.loggerInterface = loggerInterface;
+    }
+
+    @Override
+    public void log(LogLevelEnum level, String message) {
+        for(LoggerInterface loggerInterface: this.loggerInterface){
```

Comment:
`this.` se většinou nepoužívá, pokud není ve *scope* není proměnná stejného názvu (např. jako argument, popř. lokální proměnná).